monitor/server.go

```diff
@@ -1,5 +1,7 @@
 package monitor
 
+//lint:file-ignore SA1006 Allow printf-style function with dynamic format string without arguments
```

Comment:
Same here, but as this is more complex, you can do it in a separate commit or just let it be. But it would just be moving the fmt.Fprintf to fmt.Fprintln in most of the places that use formatResponse.